.github/workflows/unittests.yml

```
@@ -0,0 +1,30 @@
+name: Unit Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 10 * * 3,6'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

You probably want to use the current image here:
```suggestion
    runs-on: ubuntu-latest
```